### PR TITLE
doc: update reademe 

### PR DESCRIPTION
**Pull request by msgui**
add minor changes

### CODE REVIEW

**Review by imbajin — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -42,10 +42,12 @@ The [project homepage](https://hugegraph.apache.org/docs/quickstart/) contains m
...
-- Feedback Email: [dev@hugegraph.apache.org](mailto:dev@hugegraph.apache.org)
-- WeChat public account: Apache HugeGraph, welcome to scan this QR code to follow us.
+---
 
-<img src="https://raw.githubusercontent.com/apache/incubator-hugegraph-doc/master/assets/images/weixin.png" alt="WeChat">
+ - [GitHub Issues](https://github.com/apache/incubator-hugegraph-toolchain/issues): Feedback on usage issues and functional requirements (quick response)
+ - Feedback Email: [dev@hugegraph.apache.org](mailto:dev@hugegraph.apache.org) ([subscriber](https://hugegraph.apache.org/docs/contribution-guidelines/subscribe/) only)
+ - WeChat public account: Apache HugeGraph, welcome to scan this QR code to follow us.
+
+ <img src="https://raw.githubusercontent.com/apache/incubator-hugegraph-doc/master/assets/images/weixin.png" alt="QR png" width="350"/>
```
Comment (liuxiaocs7) [inline README.md:53]: should we use `wechat.png` instead?

Comment (imbajin) [reply README.md:53]: > should we use `wechat.png` instead?

👍🏻 nice correction, I missed it

**Review by imbajin — CHANGES_REQUESTED** [review]
update the EN pic reference (and maybe rename the old one

File: README.md [hunk 2]
```diff
@@ -42,10 +42,12 @@ The [project homepage](https://hugegraph.apache.org/docs/quickstart/) contains m
...
-- Feedback Email: [dev@hugegraph.apache.org](mailto:dev@hugegraph.apache.org)
-- WeChat public account: Apache HugeGraph, welcome to scan this QR code to follow us.
+---
 
-<img src="https://raw.githubusercontent.com/apache/incubator-hugegraph-doc/master/assets/images/weixin.png" alt="WeChat">
+ - [GitHub Issues](https://github.com/apache/incubator-hugegraph-toolchain/issues): Feedback on usage issues and functional requirements (quick response)
+ - Feedback Email: [dev@hugegraph.apache.org](mailto:dev@hugegraph.apache.org) ([subscriber](https://hugegraph.apache.org/docs/contribution-guidelines/subscribe/) only)
+ - WeChat public account: Apache HugeGraph, welcome to scan this QR code to follow us.
+
+ <img src="https://raw.githubusercontent.com/apache/incubator-hugegraph-doc/master/assets/images/wechat.png" alt="QR png" width="350"/>
```
Comment (imbajin) [inline README.md:53]: add a empty line in the end of file

**Review by imbajin — APPROVED** [review]

**Review by liuxiaocs7 — APPROVED** [review]

**Review by JackyYangPassion — APPROVED** [review]

**Pull request merged**